test/tasks.interactive.js

```diff
@@ -171,7 +171,9 @@ test.serial('should run "after:*:release" plugin hooks', async t => {
     project,
     body: {
       name: 'Release 1.1.0',
+      ref: 'master',
```

Comment:
```suggestion
      ref: isCI ? 'HEAD' : 'main',
```

Reply:
Where does `isCI` come from?  I don't see that as being a valid variable.  I do see a function of that name buried in various `config` objects in the test, not not directly as a free-standing variable.

Reply:
Oh dang, auto-import when I type in IDE:

```
import { isCI } from 'ci-info';
```

Reply:
Because this appears to create a new "bare" repo and use that as the basis for further testing, I think this one will always be "master" no matter what.  It is only the `gitlab.js` tests that having this problem because it doesn't test within a temporary bare repo.

Reply:
Normally I'd push to your branch but somehow that isn't working now.